tests/testthat/test-forecasters-data.R

```diff
@@ -1,6 +1,6 @@
-devtools::load_all()
-library(testthat)
-library(dplyr)
+for (file in list.files(here::here("R"), recursive = TRUE, full.names = TRUE)) source(file)
+
+testthat::skip("Optional, long-running tests skipped.")
```

Comment:
is this how skip works? I thought it had to be inside a `test_that`

Reply:
Turns out it just skips every test it encounters below the skip. You can see it in CI https://github.com/cmu-delphi/exploration-tooling/actions/runs/8994625447/job/24708358682#step:6:72